src/app/students/(table)/student-table-filter.tsx

```diff
@@ -0,0 +1,139 @@
+import { CheckIcon, PlusCircledIcon } from "@radix-ui/react-icons"
+import { Column } from "@tanstack/react-table"
+
+import { cn } from "@/lib/utils"
+import { Badge } from "@/components/ui/badge"
+import { Button } from "@/components/ui/button"
+import {
+    Command,
+    CommandEmpty,
+    CommandGroup,
+    CommandInput,
+    CommandItem,
+    CommandList,
+    CommandSeparator,
+} from "@/components/ui/command"
+import {
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+} from "@/components/ui/popover"
+import { Separator } from "@/components/ui/separator"
+import { useStudents } from "../(hooks)/useStudents"
+import { useState } from "react"
+
+type DataTableFacetedFilterProps<TData, TValue> = {
```

Comment:
nit;

this type has the "it is defined in another file" vibe, no need to redefine it here if that is the case